src/telemetry/telemetry.controller.ts

```diff
@@ -80,7 +80,7 @@ const TELEMETRY_WHITELIST = new Map<string, true | Array<string>>([
   ],
   ['transaction_propagation', true],
   ['forks_count', true],
```

Comment:
Was this intentional removing the `chain_database_size`?

Reply:
yes, removing it and created a new metric. The new metric is submitted from stats cmd from api-stats instance. The old one is in sdk and we decide its better to use stats cmd.